app/policies/organization_alliance_policy.rb

```diff
@@ -0,0 +1,11 @@
+class OrganizationAlliancePolicy < ApplicationPolicy
+  def update?
+    alliance = record
+    user.manages?(alliance.source_organization) || user.manages?(alliance.target_organization)
```

Comment:
just to understand, why do we need this? and also this repeated code could be an auxiliar function